recipe/meta.yaml

```
@@ -1,20 +1,18 @@
+{% set name = "scipy" %}
 {% set version = "1.15.3" %}
 
 package:
-  name: scipy
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/scipy/scipy/releases/download/v{{ version }}/scipy-{{ version }}.tar.gz
+  - url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
     sha256: eae3cf522bc7df64b42cad3925c876e1b0b6c35c1337c93e12c0f366f55b0eaf
-    patches:                                                     # [s390x]
-      - patches/0002-arff-nodata-test-remove-endian-check.patch  # [s390x]
 
 build:
   number: 0
```

```suggestion
  number: 1
```
The build number must be incremented for a rebuild.